src/Donations/Micros/PaymentStatus/SuccessfulDonation.tsx

```diff
@@ -45,7 +43,11 @@ function SuccessfulDonation({ donation, sendToReturn }: any) {
         projectDetails={projectDetails}
         donation={donation}
       />
-      <ImageComponent projectDetails={projectDetails} donation={donation} />
+      <ImageComponent
+        projectDetails={projectDetails}
+        donation={donation}
+        imageRef={imageRef}
```

Comment:
Too much code reformatting to easily review this patch 🥲 Is this the only real change?

Reply:
Yes. The ref was not being passed, because of which the download was failing.